src/sql/workbench/contrib/connection/browser/connectionStatus.ts

```diff
@@ -59,16 +63,50 @@ export class ConnectionStatusbarItem extends Disposable implements IWorkbenchCon
 	// Update the connection status shown in the bar
 	private _updateStatus(): void {
 		let activeConnection = TaskUtilities.getCurrentGlobalConnection(this._objectExplorerService, this._connectionManagementService, this._editorService, this._logService);
+		let id = undefined;
```

Comment:
nit: I'd suggest making this even clearer and rename it to `connectionId` (and it's usually nice to put the type too)